encrypted-media-respec.html

```diff
@@ -1338,7 +1338,7 @@
           <p>The following step is run:</p>
           <ol>
             <li>
-              <p>For each <a>MediaKeySession</a> created by the <var>media keys</var> on which the <a def-id="session-close-algorithm"></a> algorithm has not been run, run the <a def-id="session-close-algorithm"></a> algorithm.</p>
+              <p>For each <a>MediaKeySession</a> created by the <var>media keys</var> on which the <a def-id="session-closed-algorithm"></a> algorithm has not been run, run the <a def-id="close-session-algorithm"></a> algorithm.</p>
```

Comment:
See my [comment on the Issue](https://github.com/w3c/encrypted-media/issues/181#issuecomment-218508687).